apps/frontend/core-dapp/src/features/simulator/TradeSimulator.tsx

```diff
@@ -2,14 +2,15 @@ import { useEffect, useMemo } from 'react'
 import { observer } from 'mobx-react-lite'
 import styled from 'styled-components'
 import {
+  Button,
   Flex,
   Icon,
   mapDistanceToPrice,
   mapPriceToDistance,
   media,
   spacingIncrement,
-  Tooltip,
-} from 'prepo-ui'
+  Tooltip
+} from "prepo-ui"
```

Comment:
Is this changed by eslint? Our app uses single quote (enforced by eslint and prettier), it's weird that all other packages is still using single quote except for this one?

```suggestion
} from 'prepo-ui'
```

Reply:
So the problem I had is that my git hooks weren't running for some reason. A simple `husky install` fixed it, so I shouldn't have these kind of problems moving forward.